### PR TITLE
Verify color support

### PR DESCRIPTION
**Pull request by christophermanahan**
### Description

Allow colorizer to check if terminal supports colors
Adjust colors to match client's brand

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ Metrics/MethodLength:
   Exclude:
     - 'lib/main.rb' 
 
+Metrics/LineLength:
+  Exclude:
+    - 'lib/main.rb'
```
Comment (damonkelley) [inline .rubocop.yml]: 🤔

File: lib/app.rb [hunk 2]
```diff
@@ -1,13 +1,15 @@
 require './lib/main'
 
 OPEN_READ_WRITE = 1
-CLEAR_CONSOLE = "\e[H\e[2J".freeze
+CLEAR_CONSOLE = `clear`.freeze
```
Comment (damonkelley) [inline lib/app.rb:4]: What is the motivation behind changing this to `clear`?

**Review by damonkelley — APPROVED** [review]

File: lib/colorizer.rb [hunk 3]
```diff
@@ -1,13 +1,29 @@
...
+  def position(string)
+    colorize(string, 33)
+  end
+
+  private
+
+  attr_reader :color_count
+
+  def supported?
+    color_count >= 8
```
Comment (damonkelley) [inline lib/colorizer.rb]: What does 8 mean? Why is it significant?

File: lib/app.rb (hunk 2 above)
Comment (christophermanahan) [reply lib/app.rb:4]: I thought it was more expressive than the ANSI code.

File: lib/colorizer.rb [hunk 4]
```diff
@@ -1,13 +1,29 @@
...
+  end
+
+  def colorize(string, code)
+    supported? ? "\e[#{code}m#{string}\e[0m" : string
```
Comment (damonkelley) [inline lib/colorizer.rb]: This is doesn't look very friendly as just a literal in the body of this method. How could we communicate what this does more effectively?

File: spec/colorizer_spec.rb [hunk 5]
```diff
@@ -1,19 +1,25 @@
...
-  it 'colors a string yellow' do
-    expect(colorizer.yellow(test_string)).to eq "\e[33m#{test_string}\e[0m"
+  it 'colors symbol_2 if supported' do
+    expect(colorizer_supported.symbol_2(test)).to eq "\e[36m#{test}\e[0m"
   end
 
-  it 'colors a string cyan' do
-    expect(colorizer.cyan(test_string)).to eq "\e[36m#{test_string}\e[0m"
+  it 'colors a position if supported' do
+    expect(colorizer_supported.position(test)).to eq "\e[33m#{test}\e[0m"
```
Comment (damonkelley) [inline spec/colorizer_spec.rb]: I might opt for using a context to differentiate the specs for supported and unsupported color. You could move the relevant lets to those contexts as well.

**Pull request merged**